lib/widgets/list_cocktail.dart

```diff
@@ -3,7 +3,7 @@ import 'package:flutter/material.dart';
 import 'package:boilerplate/classes/entities/category.dart';
 
 class ListCocktail extends StatelessWidget {
-  final List<CocktailItem> list;
+  final List<TextButton> list;
```

Comment:
Is there any special reason why we change this from the `CocktailItem` for a `TextButton`? I find it more useful that if in the future we need to make more modifications or change the complete widget UI, but using the same properties of _Cocktails_, it might be more scalable to have a Widget enclosing this solution or open the type of the list for a more general one, like `Widget`, which TextButton and all the other UI inherits from and ListView will accept too. So if it a not a button anymore, and it changes to a special container  with some special decoration, we could easy just change the source without touching this file anymore